receiver/hostmetricsreceiver/internal/scraper/networkscraper/network_scraper.go

```diff
@@ -114,10 +115,10 @@ func (s *scraper) scrapeAndAppendNetworkCounterMetrics(metrics pdata.MetricSlice
 	if len(ioCounters) > 0 {
 		startIdx := metrics.Len()
 		metrics.Resize(startIdx + networkMetricsLen)
-		initializeNetworkPacketsMetric(metrics.At(startIdx+0), networkPacketsDescriptor, startTime, now, ioCounters)
-		initializeNetworkDroppedPacketsMetric(metrics.At(startIdx+1), networkDroppedPacketsDescriptor, startTime, now, ioCounters)
-		initializeNetworkErrorsMetric(metrics.At(startIdx+2), networkErrorsDescriptor, startTime, now, ioCounters)
-		initializeNetworkIOMetric(metrics.At(startIdx+3), networkIODescriptor, startTime, now, ioCounters)
+		initializeNetworkPacketsMetric(metrics.At(startIdx+0), metadata.Metrics.SystemNetworkPackets.New(), startTime, now, ioCounters)
+		initializeNetworkDroppedPacketsMetric(metrics.At(startIdx+1), metadata.Metrics.SystemNetworkDropped.New(), startTime, now, ioCounters)
+		initializeNetworkErrorsMetric(metrics.At(startIdx+2), metadata.Metrics.SystemNetworkErrors.New(), startTime, now, ioCounters)
+		initializeNetworkIOMetric(metrics.At(startIdx+3), metadata.Metrics.SystemNetworkIo.New(), startTime, now, ioCounters)
```

Comment:
I think these could be slightly refactored to work like the other ones where Init is called instead of doing New and then CopyTo.

Reply:
Updated.